test/Microsoft.DotNet.SourceBuild.Tests/assets/SdkContentTests/ValidateAssemblies.proj

```diff
@@ -0,0 +1,67 @@
+<Project Sdk="Microsoft.Build.NoTargets">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <UpdatedSuppressionFilePath>$(ArtifactsTestResultsDir)generated_ApiDiff.suppression</UpdatedSuppressionFilePath>
+    <BaselineSuppressionFilePath>$(MSBuildThisFileDirectory)ApiDiff.suppression</BaselineSuppressionFilePath>
+    <TestRoot>$([System.IO.Path]::GetTempPath())$([System.IO.Path]::GetRandomFileName())</TestRoot>
+    <MsftPath>$(TestRoot)/msft</MsftPath>
+    <SbPath>$(TestRoot)/sb</SbPath>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageVersion Include="Microsoft.DotNet.ApiCompat.Task" Version="10.0.100-rc.1.25451.107" />
+    <PackageVersion Include="Microsoft.CodeAnalysis" Version="4.4.0" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.DotNet.ApiCompat.Task" />
+    <PackageReference Include="Microsoft.CodeAnalysis" />
+  </ItemGroup>
+
+  <UsingTask TaskName="ValidateAssembliesTask"
+             AssemblyFile="$(PkgMicrosoft_DotNet_ApiCompat_Task)/tools/net/Microsoft.DotNet.ApiCompat.Task.dll" />
+
+  <Target Name="RunApiCompat" DependsOnTargets="Restore" BeforeTargets="Build">
+    <MakeDir Directories="$(MsftPath)" Condition="!Exists('$(MsftPath)')" />
+    <MakeDir Directories="$(SbPath)" Condition="!Exists('$(SbPath)')" />
+
+    <Exec Command="tar -xzf $(MsftSdkTarballPath) -C $(MsftPath)" />
```

Comment:
In the spirit of fast tests, it looks as those the tests will not extract these tarballs multiple times (here and in CompareMsftToSbAssemblyVersions) which is unfortunate.